rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java

```diff
@@ -39,6 +39,14 @@ private InformixUtil() {
     public static final String TABLESPACE_UTILIZATION_SQL = "select case WHEN (pt.nptotal > 0) THEN ((pt.npused) /pt.nptotal) * 100 ELSE 0 END AS table_utilization, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
     public static final String TABLESPACE_MAX_SQL = "SELECT(pt.nptotal * pt.pagesize)  * 1024 AS total_kb, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
 
+    public static final String DB_DISK_WRITE_COUNT_SQL = "SELECT value From sysprofile where name = 'dskwrites'";
+    public static final String DB_DISK_READ_COUNT_SQL = "SELECT value From sysprofile where name = 'dskreads'";
+
+    public static final String DB_DATABASE_LOG_ENABLED_SQL = "SELECT is_logging FROM sysdatabases";
+    public static final String DB_DATABASE_BUFF_LOG_ENABLED_SQL = "SELECT is_buff_log FROM sysdatabases";
```

Comment:
Please have a look at the semantic over [here](https://github.com/instana/otel-dc/blob/main/docs/semconv/database.md#settings-metrics)
each metrics has a mandatory attribute i.e. `database_name`